test/fixtures/flow/type-annotations/136/actual.js

```diff
@@ -0,0 +1,4 @@
+type A = {
+	p: {},
+	...{},
+};
```

Comment:
Oh okay sorry. I see astexplorer uses flow 0.40 which supports it spreading only for Identifiers?